front/src/components/group/groupHeader/index.js

```diff
@@ -17,6 +17,17 @@ class GroupHeader extends Component {
         avatar: this.props.currentGroup.avatar,
     }
 
+
+    componentDidUpdate(prevProps, prevState, snapshot) {
+        if(prevProps.currentGroup.id !== this.props.currentGroup.id) {
```

Comment:
Please, add check that `prevProps.currentGroup` and `this.props.currentGroup`are not `undefined`

Reply:
done